app/models/communitarian/resolution_title.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Communitarian
+  class ResolutionTitle
+    def self.create_suffix_for!(topic)
+      new_title = new(topic).to_s
+      topic.update!(title: new_title)
+    end
+
+    def initialize(topic)
+      @topic = topic
+    end
+
+    def to_s
+      "#{topic.title} - #{cateogry_abbreviation} #{topic.id}"
+    end
+
+    private
+
+    attr_reader :topic
+
+    def cateogry_abbreviation
```

Comment:
we should use value from community code column.
```suggestion
    def category_abbreviation
```